plasmapy/formulary/frequencies.py

```diff
@@ -308,32 +322,21 @@ def plasma_frequency(n: u.m**-3, particle: ParticleLike, z_mean=None) -> u.rad /
     this function and can be used as follows.
 
     >>> from plasmapy.particles import Particle
-    >>> mass = Particle("p").mass.value
-    >>> plasma_frequency.lite(n=1e19, mass=mass, z_mean=1)
+    >>> mass = Particle("p+").mass.value
+    >>> plasma_frequency.lite(n=1e19, mass=mass, Z=1)
     416329...
-    >>> plasma_frequency.lite(n=1e19, mass=mass, z_mean=1, to_hz=True)
+    >>> plasma_frequency.lite(n=1e19, mass=mass, Z=1, to_hz=True)
     662608...
     """
-
-    try:
-        m = particles.particle_mass(particle).value
-
-        if z_mean is None:
-            # warnings.warn("No z_mean given, defaulting to atomic charge",
-            #               PhysicsWarning)
-            try:
-                Z = particles.charge_number(particle)
-            except ChargeError:
-                Z = 1
```

Comment:
This PR removes the assumption that the charge number of an ion defaults to 1 if not provided.

Reply:
Good - assumptions are bad!